source/guide_redis.rst

```diff
@@ -0,0 +1,189 @@
+.. highlight:: console
+
+.. author:: Nico Graf <hallo@uberspace.de>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/redis.svg
+      :align: center
+
+##########
+Redis
+##########
+
+Redis_ is a key-value store NoSQL database. It is primarily used because of its high read and write rates. Redis uses two columns, storing pairs of one key and one corresponding value.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of supervisord_.
+
+
+License
+=======
+
+Redis is distributed under the `BSD license`_.
+
+Installation
+============
+
+Download and extract Redis
+--------------------------
+
+.. code-block:: bash
+
+ [isabell@stardust ~]$ wget http://download.redis.io/redis-stable.tar.gz -O ~/redis-stable.tar.gz
+ --2019-01-03 13:51:21--  http://download.redis.io/redis-stable.tar.gz
+ Resolving download.redis.io (download.redis.io)... 109.74.203.151
+ Connecting to download.redis.io (download.redis.io)|109.74.203.151|:80... connected.
+ HTTP request sent, awaiting response... 200 OK
+ Length: 1999982 (1.9M) [application/x-gzip]
+ Saving to: ‘/home/isabell/redis-stable.tar.gz’
+
+ 100%[======================================>] 1,999,982   --.-K/s   in 0.07s
+
+ 2019-01-03 13:51:21 (27.3 MB/s) - ‘redis-stable.tar.gz’ saved [1999982/1999982]
+ [isabell@stardust ~]$ tar xfv redis-stable.tar.gz
```

Comment:
May I suggest to split the first code block in download and untar to improve readability.